modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase

```diff
@@ -508,6 +508,33 @@ definition {
 		}
 	}
 
+	@description = "LPS-164948 - Verify that the user is able to add formula fields when managing System Objects"
+	@priority = 5
+	test CanCreateFormulaFieldOnSystemObject {
+		property portal.upstream = "true";
```

Comment:
This property is defined in the testcase definition. I think you meant portal.acceptance